test/alternat_test.go

```diff
@@ -0,0 +1,311 @@
+package test
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	ec2types "github.com/aws/aws-sdk-go-v2/service/ec2/types"
+
+	terraws "github.com/gruntwork-io/terratest/modules/aws"
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/gruntwork-io/terratest/modules/retry"
+	"github.com/gruntwork-io/terratest/modules/ssh"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	test_structure "github.com/gruntwork-io/terratest/modules/test-structure"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAlternat(t *testing.T) {
+	// os.Setenv("SKIP_setup", "true")	
+	// os.Setenv("SKIP_apply_vpc", "true")	
+	// os.Setenv("SKIP_apply_alternat_basic", "true")	
+	// os.Setenv("SKIP_validate_alternat_basic", "true")	
+	// os.Setenv("SKIP_validate_alternat_setup", "true")	
+	// os.Setenv("SKIP_validate_alternat_replace_route", "true")	
+	// os.Setenv("SKIP_cleanup", "true")	
```

Comment:
Do you want to leave this here?

Reply:
Added a comment explaining this in 8e30199